src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -80,8 +80,8 @@ public void should_idempotently_create_case_from_valid_exception_record() throws
         assertThat(createdCase.getData().get("lastName")).isEqualTo("value2");
         assertThat(createdCase.getData().get("email")).isEqualTo("hello@test.com");
 
-        assertThat(createdCase.getData().get(EXCEPTION_RECORD_REFERENCE)).isNotNull();
-        String caseExceptionRecordReference = (String) createdCase.getData().get(EXCEPTION_RECORD_REFERENCE);
+        assertThat(createdCase.getData().get(BULK_SCAN_CASE_REFERENCE_FIELD)).isNotNull();
```

Comment:
you can use existing var `caseExceptionRecordReference `

Reply:
True. I was just replacing constants in this class, but will make this change, too.